test/components/remote_retro_test.js

```diff
@@ -32,4 +39,28 @@ describe("RemoteRetro component", () => {
       expect(hotjarSpy).calledWith("trigger", CLOSED)
     })
   })
+
+  describe("when component updates with a new facilitator", () => {
+    let newFacilitator
+    beforeEach(() => {
+      newFacilitator = {
+        ...stubUser,
+        is_facilitator: true,
+      }
+    })
+
+    it("calls the new facilitator action", () => {
+      const wrapper = shallow(<RemoteRetro {...defaultProps} />)
+      wrapper.setProps({ user: newFacilitator })
+      expect(defaultProps.actions.newFacilitator.calledWith(defaultProps.retro))
```

Comment:
@edem-ca hey bud! just a heads up that this was always passing, do to there being no assertion attached to the expect. I removed the passage of `retro` in the implementation, and it was still green. Again, this surfaces a need for you to test drive your changes, as that would have made the false positive apparent!

Reply:
also! there was an extension of the false positive issue, in that the `setProps` call wasn't actually triggering the code in question, due to passing `user` instead of `currentUser`